libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -34,17 +39,23 @@ ControlFlowBuilder::ControlFlowBuilder(CFG::NodeContainer& _nodeContainer, Funct
 {
 }
 
-
-unique_ptr<FunctionFlow> ControlFlowBuilder::createFunctionFlow(
-	CFG::NodeContainer& _nodeContainer,
-	FunctionDefinition const& _function
-)
+std::unique_ptr<FunctionFlow> ControlFlowBuilder::initFunctionFlow(CFG::NodeContainer& _nodeContainer)
 {
 	auto functionFlow = make_unique<FunctionFlow>();
 	functionFlow->entry = _nodeContainer.newNode();
 	functionFlow->exit = _nodeContainer.newNode();
 	functionFlow->revert = _nodeContainer.newNode();
 	functionFlow->transactionReturn = _nodeContainer.newNode();
+	return functionFlow;
+}
+
+unique_ptr<FunctionFlow> ControlFlowBuilder::createFunctionFlow(
+	CFG::NodeContainer& _nodeContainer,
+	FunctionDefinition const& _function
+)
+{
+	std::unique_ptr<FunctionFlow> functionFlow = initFunctionFlow(_nodeContainer);
+	map<FunctionDefinition const*, unique_ptr<FunctionFlow>> visitedFunctions;
```

Comment:
What is this line doing?

Reply:
Left over from a refactoring, will remove